api/controllers/countries.js

```diff
@@ -19,7 +19,8 @@ var resultsQuery = db
 var activeQuery = db.select(db.raw(`* from pg_stat_activity where state = 'active' and query = '${resultsQuery.toString()}'`));
 
 if (process.env.USE_ATHENA) {
-  resultsQuery = client.query('SELECT country, city, location, count(location) as count from fetches.fetches_realtime GROUP BY country, city, location ORDER BY country');
+  let query = 'SELECT country, city, location, count(location) as count from ' + client.fetchesTable + ' GROUP BY country, city, location ORDER BY country';
```

Comment:
You can use JS template literals if you want, might read more nicely than this way.

Reply:
Oddly enough I tried them but kept getting lint errors.

Reply:
Huh, that's weird. Um, if you want to put them back in, I can fix up the linting errors.

Reply:
I've added template strings, but couldn't get multiline template strings to work...